Dockerfile.graphd

```diff
@@ -0,0 +1,23 @@
+FROM vesoft/nebula-dev:centos as builder
+
+LABEL maintainer "yee.yi@vesoft.com"
+
+COPY . /home/nebula/BUILD
+
+RUN cd /home/nebula/BUILD/package \
+  && . /etc/profile.d/devtoolset-8.sh \
+  && ./make_srpm.sh -v $(git rev-parse --short HEAD) -p /home/nebula graphd
+
+FROM centos:7
+
+COPY --from=builder /home/nebula/RPMS/x86_64/nebula-graphd-*.el7.x86_64.rpm /usr/local/nebula/nebula-graphd.rpm
```

Comment:
we need the common rpm, it is nebula-*.el7.centos.x86_64.rpm

Reply:
We want to run single service in one container. So only need to copy the specified rpm file.

Reply:
The three services have something in common, such as a shared script, they are packed together in nebula-${version}.el7.centos.x86_64.rpm

Reply:
Got it！